packages/contracts-bedrock/interfaces/L1/IOPContractsManager.sol

```diff
@@ -216,10 +218,10 @@ interface IOPContractsManager {
 
     function deploy(DeployInput calldata _input) external returns (DeployOutput memory);
 
-    /// @notice Upgrades a set of chains to the latest implementation contracts
-    /// @param _opChains Array of OpChain structs, one per chain to upgrade
-    /// @dev This function is intended to be called via DELEGATECALL from the Upgrade Controller Safe
-    function upgrade(OpChain[] memory _opChains) external;
+    /// @notice Upgrades the implementation of all proxies in the specified chains
+    /// @param _superchainProxyAdmin The proxy admin that owns all of the proxies
```

Comment:
nit, since this comment confused me at first
```suggestion
    /// @param _superchainProxyAdmin The proxy admin that owns the superchain contract proxies
```